_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,569 @@
+    reduces the urgency of deploying a quantum-resistant scheme in the
+    short term.
+
+    Lloyd Fournier [says][fournier gfsig], "if this approach gains
+    acceptance I think the main immediate action users can take is to
+    move to a taproot wallet" because of its ability to allow keypath
+    spending under the current consensus rules, including in the case of
+    [address reuse][topic output linking], but also resistance to
+    quantum theft if keypath spending is later disabled.
+
+    In a separate thread (see next item), Pieter Wuille [notes][wuille
+    nonpublic] that UTXOs vulnerable to quantum theft also includes keys
+    which have not been used publicly but which are known to multiple
+    parties, such as in various forms of multisig (including LN,
+    [DLCs][topic dlc], and escrow services).
+
+  - *Draft BIP for destroying quantum-insecure bitcoins:* Agustin Cruz
+    [posted][cruz qramp] to the Bitcoin-Dev mailing a [draft BIP][cruz
+    bip] that describes several options for a general process of
+    destroying bitcoins that are vulnerable to quantum theft (in the
+    event that becomes an expected risk).  Cruz argues, "enforcing a
+    deadline for migration, we provide rightful owners with a clear,
+    non-negotiable opportunity to secure their funds [...] a forced
+    migration, with sufficient notice and robust safeguards, is both
+    realistic and necessary to protect the long-term security of
+    Bitcoin."
+
+    Very little of the discussion on the thread focused on the draft
+    BIP.  Most of it focused on whether or not destruction of quantum
+    vulnerable bitcoins was a good idea, similar to the thread later
+    started by Jameson Lopp (described in a previous sub-item).
+
+- **Multiple discussions about a CTV+CSFS soft fork:** several
+  conversations examined various aspects of soft forking in the
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV) and
+  [OP_CHECKSIGFROMSTACK][topic op_checksigfromstack] (CSFS) opcodes.
+
+  - *Criticism of CTV motivation:* Anthony Towns [posted][towns ctvmot]
+    a criticism of [BIP119][]'s described motivation for CTV, a
+    motivation which he argued would be undermined by adding both CTV
+    and CSFS to Bitcoin.  Several days after the discussion started,
+    BIP119 was updated by its author to remove most (and possibly all)
+    of the controversial language; see [Newsletter #347][news347 bip119]
+    for our summary of the change and the [older version][bip119
+    prechange] for BIP119 for reference.  Some of the topics discussed
+    included:
+
+    - *CTV+CSFS allows creation of a perpetual covenant:*
+      CTV's motivation said, "Covenants have historically been widely
+      considered to be unfit for Bitcoin because they are too complex to
+      implement and risk reducing the fungibility of coins bound by
+      them.  This BIP introduces a simple covenant called a *template*
```

Comment:
```suggestion
      them.  This BIP introduces a simple covenant called a *template*,
```

Reply:
Leaving as-is.  Quoted text; I sometimes slightly clean up punctuation in quotes but here we make a big deal about preserving the emphasis in the quote, so I want it verbatim.